test/api/v3/integration/user/DELETE-user.test.js

```diff
@@ -30,6 +30,20 @@ describe('DELETE /user', () => {
     });
   });
 
+  it('does not need a password if logged in through Facebook', async () => {
+    await user.auth.facebook.id.del('/user', {
```

Comment:
Thanks, we are trying that and the test comes back with this error:
`path `auth.facebook.id` was not saved, as it's a protected path.`

Here is the updated test code: 

```  it('does not need a password if logged in through Social Network to delete account', async () => {
    await user.update({
      'auth.facebook.id': 'some-fb-id',
      'auth.google.id': 'some-google-id',
      'auth.local': { ok: true },
    });

    await user.del('/user', {
    });

    await user.sync();
    await expect(checkExistence('users', user._id)).to.eventually.eql(false);
  });```